doc/reference/dev/debug.md

```diff
@@ -1,5 +1,130 @@
 # Debugging
 
-(TODO)
+Complex programs usually start as scribbling in a buffer, evaluated in
+the REPL. The scribbling then becomes a module, which later becomes a
+library, and eventually the program takes its more or less final
+shape. In the end, it is compiled as a fully static binary and shipped
+to a production server.
 
-See [debugging libraries](/reference/std/debug.md) for now.
+One thing is certain: in all stages of a program's evolution there
+will be bugs.  Gerbil offers various mechanisms for debugging
+programs, from inception to production.
+
+See also [Testing](test.md).
+
+## Debugging in the REPL
+
```

Comment:
A note is needed along the lines of:

You can `(load "foo.ss")` the source of the module if it doesn't have relative imports such as `(import ./bar)`, but more importantly, you can `(import :pkg/foo)` then `(enter! :pkg/foo)`.

Probably in its own section.

Reply:
Dunno, it's kind of documented elsewhere, and it is implicit. Not sure it adds anything but clutter.

Reply:
I think I'll add a reference to the interactive page in stdlib ref, this should be sufficient.

Reply:
done, it's in the guide actually.